extensions/data-plane/data-plane-aws-s3/src/main/java/org/eclipse/edc/connector/dataplane/aws/s3/DataPlaneS3Extension.java

```diff
@@ -29,6 +30,12 @@
 public class DataPlaneS3Extension implements ServiceExtension {
 
     public static final String NAME = "Data Plane S3 Storage";
+
+    static final int DEFAULT_CHUNK_SIZE_IN_MB = 500; // 500MB chunk size
+
+    @Setting
```

Comment:
please add details to the `@Setting` annotation, like `value`, `defaultValue`, `type` and so on